### PR TITLE
Add proxy and dry-run false to the blocking issue creator config

### PR DESCRIPTION
**Pull request by jmguzik**
Related to the changes made in https://github.com/openshift/ci-tools/pull/2484
Probably this one should be merged before.

Signed-off-by: Jakub Guzik <jguzik@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -1320,6 +1320,11 @@ periodics:
   spec:
     containers:
     - args:
+      - --github-token-path=/etc/github/oauth
```
Comment (petr-muller) [inline ci-operator/jobs/infra-periodics.yaml:1323]: Not this one, it's already at the end of the list (but it makes sense to move it here)

**Review by petr-muller — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/infra-periodics.yaml (hunk 1 above)
Comment (droslean) [reply ci-operator/jobs/infra-periodics.yaml:1323]: Perhaps it's a nice opportunity to create a new token with more restricted permissions.

Comment (jmguzik) [reply ci-operator/jobs/infra-periodics.yaml:1323]: agh, so stupid mistake, haven't noticed

Comment (jmguzik) [reply ci-operator/jobs/infra-periodics.yaml:1323]: ... but corrected now

**Review by droslean — APPROVED** [review]

**Pull request merged**